src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -3361,6 +3362,42 @@ class misc_commandscript : public CommandScript
         handler->GetSession()->SendShowMailBox(player->GetGUID());
         return true;
     }
+
+    static bool HandleStringCommand(ChatHandler *handler, char const *args)
+    {
+        if (!*args)
+        {
+            handler->SendSysMessage(LANG_CMD_SYNTAX);
+            return false;
+        }
+
+        uint32 id = atoi(strtok((char*)args, " "));
+        if(id == 0)
+        {
+            handler->SendSysMessage(LANG_CMD_SYNTAX);
+            return false;
+        }
+
+        uint32 locale = 0;
+        char* localeString = strtok(nullptr, " ");
+        if(localeString != nullptr)
+        {
+            locale = atoi(localeString);
+        }
+
+        const char* str = sObjectMgr->GetAcoreString(id, static_cast<LocaleConstant>(locale));
+
+        if(strcmp(str, "<error>") == 0)
```

Comment:
```suggestion
        if (id == 0)
        {
            handler->SendSysMessage(LANG_CMD_SYNTAX);
            return false;
        }

        uint32 locale = 0;
        char* localeString = strtok(nullptr, " ");
        if (localeString != nullptr)
        {
            locale = atoi(localeString);
        }

        const char* str = sObjectMgr->GetAcoreString(id, static_cast<LocaleConstant>(locale));

        if (strcmp(str, "<error>") == 0)
```

Reply:
Sorry I did not see this. I'm going to update my CLion style config.